### ISSUE TITLE
JavaScript error with Demo 2

### ISSUE THREAD

**Issue by funkatron82** [ocean90/media-modal-demo#3]
I'm getting a weird JS console error when I press the button in demo 2:

```
Uncaught ReferenceError: wpActiveEditor is not defined load-scripts.php:6
wp.media.editor.id load-scripts.php:6
wp.media.editor.open load-scripts.php:6
mediaUploaderDemo.openMediaDialog media-modal-demo-2.js:17
v.event.dispatch load-scripts.php:2
o.handle.u
```

**Comment by ocean90** [comment]
Odd, do you have any other plugins active?

**Comment by funkatron82** [comment]
I have a fiew.  Let me disable all of them and see if it works then.
 Didn't think about other plugins interfering.

On Mon, Feb 25, 2013 at 1:56 PM, Dominik Schilling <notifications@github.com

**Comment by funkatron82** [comment]
Just disabled the other 3 plugins I had.  It cut down the number of lines it gives for the error but it's still there:

```
Uncaught ReferenceError: wpActiveEditor is not defined media-modal-demo-2.js:17
mediaUploaderDemo.openMediaDialog media-modal-demo-2.js:17
v.event.dispatch load-scripts.php:2
o.handle.u load-scripts.php:2

```

**Comment by ocean90** [comment]
Okay, getting the error now too.
I had another plugin active which has added it's own scripts on every page. Because of this `wpActiveEditor` was defined.

Thanks for the report, will be fixed in 1.0.
